Update permissions.md (#8767)
DataDog/documentation: content/en/account_management/rbac/permissions.md

## Log Management

> Find below the list of permissions for the log configuration assets and log data, along with the typical category of user to whom assign this permission (refer to our recommendations on how to assign permissions to team members in the [Logs RBAC][17] guide).

Reviewer: ```suggestion
Find below the list of permissions for the log configuration assets and log data, along with the typical category of user you'd assign this permission to. See the recommendations on how to assign permissions to team members in the [Logs RBAC guide][17].
```

| [`logs_read_data`](#logs_read_data)                            | Read access to log data                    | true     | Read-Only     |
| [`logs_modify_indexes`](#logs_modify_indexes)                  | Update the definition of log indexes       | false    | Admin         |
| [`logs_write_facets`](#logs_write_facets)                      | Create, Update and Delete Log Facets       | false    | Standard      |
| [`logs_write_exclusion_filters`](#logs_write_exclusion_filters)| Update indexes exclusion filters           | true     | Standard      |
| [`logs_write_pipelines`](#logs_write_pipelines)                | Update log pipelines                       | false    | Admin         |
| [`logs_write_processors`](#logs_write_processors)              | Update the log processors in a pipeline    | true     | Standard      |
| [`logs_write_archives`](#logs_write_archives)                  | Update the external archives configuration | false    | Admin         |
| [`logs_read_archives`](#logs_read_archives)                    | See archive configuration details, access content from the archive | true     | Standard |

> | [`logs_write_historical_views`](#logs_write_historical_views)  | Rehydrate data from Archives               | false    | Standard      |

Reviewer: the heading this is pointing to is "view" so the anchor is broken. I suspect the heading should be "views" but leave it to you to figure out which

| [`logs_public_config_api`](#logs_public_config_api)            | Access the Logs Public Config API (r/w)    | false    | Admin         |
| [`logs_generate_metrics`](#logs_generate_metrics)              | Access the Generate Metrics feature        | false    | Standard      |

Log Management RBAC also includes two legacy permissions, superseded by finer-grained and more extensive `logs_read_data` permission:

| Name                                              | Description                                | Scopable |  Typical User |
| ------------------------------------------------- | ------------------------------------------ | -------- | ------------- |

> | [`logs_live_tail`](#logs-live-tail)               | Access the live tail feature               | false    | Read-Only     |

Reviewer: ```suggestion
| [`logs_live_tail`](#logs_live_tail)               | Access the live tail feature               | false    | Read-Only     |
```

> | [`logs_read_index_data`]](#logs-read-index-data)  | Read a subset log data (index based)       | true     | Read-Only     |

Reviewer: ```suggestion
| [`logs_read_index_data`](#logs_read_index_data)  | Read a subset log data (index based)       | true     | Read-Only     |
```

This permission is global and enables both the creation of new metrics, and the edition or deletion of existing ones.

#### logs_write_facets

> Grants a role the ability to use the [Create, Edit and Delete Facets][18].

Reviewer: ```suggestion
Grants a role the ability to use the [Create, Edit, and Delete facets][18].
```

This permission is global and enables both the creation of new facets, and the edition or deletion of existing ones.

> This permission has no incidence on the management of [Standard Attributes][19] or [Aliasing Facets][20].

Reviewer: ```suggestion
This permission has no effect on the management of [standard attributes][19] or [aliasing facets][20].
```

[1]: /account_management/users/#edit-a-user-s-roles
[2]: /api/v2/roles/#list-permissions
[3]: /logs/guide/logs-rbac/?tab=ui#overview
[4]: /logs/logs_to_metrics/
[5]: /logs/explorer/facets/#overview
[6]: /logs/processing/attributes_naming_convention/#standard-attributes-in-log-configuration
[7]: /logs/explorer/facets/#alias-facets
[8]: /logs/indexes
[9]: /logs/indexes#indexes-filters
[10]: /logs/indexes#update-log-retention
[11]: /logs/indexes#exclusion-filters
[12]: /logs/processing/pipelines/
[13]: /logs/processing/pipelines/#pipeline-filters
[14]: /logs/archives
[15]: /logs/archives/rehydrating

> [16]: #logs-read-archives

Reviewer: ```suggestion
[16]: #logs_read_archives
```
